src/building/how-to-build-and-run.md

```diff
@@ -277,18 +277,20 @@ default).
 Once you have successfully built `rustc`, you will have created a bunch
 of files in your `build` directory. In order to actually run the
 resulting `rustc`, we recommend creating rustup toolchains. The first
-one will run the stage1 compiler (which we built above). The second
-will execute the stage2 compiler (which we did not build, but which
-you will likely need to build at some point; for example, if you want
-to run the entire test suite).
+command listed below runs the stage1 compiler, which was built in the
+steps above, with the name `stage1`. The second command runs the stage2 
+compiler using the stage1 compiler. This will be needed in the future 
```

Comment:
more correct, maybe
```suggestion
command listed below creates the stage1 toolchain, which was built in the
steps above, with the name `stage1`. The second command creates the stage2 
toolchain using the stage1 compiler. This will be needed in the future 
```